doc/modules/ensemble.rst

```diff
@@ -98,14 +98,19 @@ controls the number of iterations of the boosting process::
   >>> clf.score(X_test, y_test)
   0.8965
 
-Available losses for regression are 'squared_error',
-'absolute_error', which is less sensitive to outliers, and
-'poisson', which is well suited to model counts and frequencies. For
-classification, 'log_loss' is the only option. For binary classification it uses the
-binary log loss, also known as binomial deviance or binary cross-entropy. For
-`n_classes >= 3`, it uses the multi-class log loss function, with multinomial deviance
-and categorical cross-entropy as alternative names. The appropriate loss version is
-selected based on :term:`y` passed to :term:`fit`.
+Available losses for **regression** are:
+
+- 'squared_error', which is the default loss;
+- 'absolute_error', which is less sensitive to outliers than the squared error;
+- 'poisson', which is well suited to model counts and frequencies;
+- 'quantile' that allows for estimating prediction intervals.
```

Comment:
Rather than stating "prediction intervals", I would rather mention conditional quantile. Basically, we can only estimate a single quantile so referring "intervals" would be confusing here.

Maybe we could state something like

```suggestion
- 'quantile' that allows for estimating conditional quantile that can later
  used to predict confidence intervals.
```